src/views/simulation/resultats.vue

```diff
@@ -129,13 +144,16 @@ export default {
     if (this.$route.query?.situationId) {
       this.$route.query.simulationId = this.$route.query.situationId
     }
-
     if (this.mock(this.$route.params.droitId)) {
       return
     } else if (this.$route.query?.simulationId) {
       if (this.store.simulationId !== this.$route.query.simulationId) {
         this.store.fetch(this.$route.query.simulationId).then(() => {
-          this.store.compute()
+          if (this.simulationAnonymized()) {
+            this.sendAccessToAnonymizedResults()
```

Comment:
En faisant un test (en partant de la forme d'url dans l'email c'et à dire /followups/:followup_id?token=:token) j'ai l'impression qu'on ne passe pas par ici car la condition ligne 150 `this.store.simulationId !== this.$route.query.simulationId` n'est pas remplie.

Je ne vois donc aucun event matomo partir

Reply:
Ok, my bad, en fait ça ne marche que la première fois (mais on veut probablement ça !), j'ai clear le cache et j'ai bien vu l'event partir : 
<img width="344" alt="image" src="https://user-images.githubusercontent.com/4059803/234866252-cf6826c0-5c9a-4c06-ab45-bcd10411fc90.png">
Donc ça me semble bien !